runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```diff
@@ -537,4 +530,35 @@ private int getHaPrice(int itemId)
 				return itemManager.getItemComposition(itemId).getHaPrice();
 		}
 	}
+
+	private ContainerPrices getContainerGEandAlchPrice(WidgetInfo widgetInfo, InventoryID inventoryID)
+	{
+		final Widget bankItemContainer = client.getWidget(widgetInfo);
+		final ItemContainer bankContainer = client.getItemContainer(inventoryID);
+		final Widget[] children = bankItemContainer.getChildren();
+		ContainerPrices prices = null;
+
+		if (bankContainer != null && children != null)
+		{
+			long geTotal = 0, haTotal = 0;
+			log.debug("Computing bank price of {} items", bankContainer.size());
+
+			// The first components are the bank items, followed by tabs etc. There are always 816 components regardless
+			// of bank size, but we only need to check up to the bank size.
```

Comment:
This comment should be updated as it no longer only runs for the bank. Prepending it with "In the bank, ..." would be sufficient.